testapps/test_elasticsearch/tests/test_prod_es_indices.py

```diff
@@ -55,7 +55,7 @@ def test_prod_config(self):
     },
     {
         "alias": "hqusers",
-        "index": "test_hqusers_2016-02-16_1402",
+        "index": "test_hqusers_2016-07-11-14-34",
```

Comment:
nope - it just needs to be unique, and to change whenever we update the mapping.